optuna/integration/mlflow.py

```diff
@@ -280,7 +301,9 @@ def _set_tags(self, trial: optuna.trial.FrozenTrial, study: optuna.study.Study)
                 tags[key] = "{}...".format(value[: max_val_length - 3])
 
         # This sets the tags for MLflow.
-        mlflow.set_tags(tags)
+        # MLflow handles up to 100 tags per request
```

Comment:
We prefer to use a comment as a full sentence.
https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences

```suggestion
        # MLflow handles up to 100 tags per request.
```